<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>250</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
    <pointsize>9</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(20, 20, 20);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>480</width>
      <height>230</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 27, 27);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 27, 27);</string>
    </property>
    <property name="text">
     <string>Include Numbers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Generate Password</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 27, 27);</string>
    </property>
    <property name="text">
     <string>Password Generator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>210</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Info</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 27, 27);</string>
    </property>
    <property name="text">
     <string>Include Small Letters</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 27, 27);</string>
    </property>
    <property name="text">
     <string>Include Special Chars</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 27, 27);
</string>
    </property>
    <property name="text">
     <string>Include Big Letters</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 20, 20);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 27, 27);
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 27, 27);</string>
    </property>
    <property name="text">
     <string>Password Length</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>17</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 27, 27);</string>
    </property>
    <property name="text">
     <string>Easy To Read</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 27, 27);</string>
    </property>
    <property name="text">
     <string>Easy To Say</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Copy Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>checkBox</zorder>
   <zorder>pushButton</zorder>
   <zorder>label</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>checkBox_2</zorder>
   <zorder>checkBox_3</zorder>
   <zorder>checkBox_4</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>comboBox</zorder>
   <zorder>checkBox_5</zorder>
   <zorder>checkBox_6</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>label_5</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
